### PR TITLE
User/emily/patch example

### PR DESCRIPTION
**Pull request by Emz-programmer**
- Updated blah blah

### CODE REVIEW

File: src/main/java/se/nymia/Dummy2.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package se.nymia;
+
+public class Dummy2 {
+    public static void doThingy(){
+        //Test yoda
+        //Another fucking commit
```
Comment (Emz-programmer) [inline src/main/java/se/nymia/Dummy2.java:6]: WTF is this braindamaged code????

File: src/main/resources/config.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+api-endpoint: DEV_ENV
```
Comment (Emz-programmer) [inline src/main/resources/config.yaml:1]: I think we need to pull this from whatever file instead of hard coding

Comment (Emz-programmer) [reply src/main/resources/config.yaml:1]: We looked into that and due to X it doesn't work as we need it to

File: src/main/java/se/nymia/Dummy2.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package se.nymia;
+
+public class Dummy2 {
+    public static void doThingy(){
+        //Test yoda
```
Comment (xNymia) [inline src/main/java/se/nymia/Dummy2.java:5]: Why you do this?

**Review by xNymia — CHANGES_REQUESTED** [review]
Change plz before 🚢

**Review by xNymia — APPROVED** [review]

**Pull request merged**